GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/GameData.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+[Serializable]
+public class GameData
+{
+    public string carInUseName;
+    public bool[] carIndexToOwnership;
+    public int coinsOwned;
+
+    private Car _carInUseObj;
+    private const int InitialCoinAmount = 20;
+    private const int TotalCarCount = 4;
+    private const bool Owned = true;
+    private const bool NotOwned = false;
```

Comment:
enum class Ownership

Reply:
is it necessary to use enum class if I can represent owned and not owned with true and false?